tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/jmx/PortalBundleDeployer.java

```diff
@@ -35,12 +35,6 @@ public void updateBundleFromURL( long bundleId, String bundleUrl ) throws Except
     {
         final ObjectName framework = getFramework(mBeanServerConnection);
 
-        mBeanServerConnection.invoke(framework, "stopBundle", new Object[] {
```

Comment:
yes, if you create fragment bundle and deploy and change some code in it , ide will rebuild and redeploy it and will call this method , but we can't do stop command on a fragment project.
Also we already stop the no-fragment bundle in https://github.com/liferay/liferay-ide/blob/master/tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundleSupervisor.java#L95-L113